locale/lang-dk.js

```diff
@@ -291,7 +291,7 @@ SnapTranslator.dict.dk = {
     "Plain prototype labels": "Klare prototypemærkater",
     "Play": "Afspil",
     "Play sound": "Afspil lyd",
-    "Please make sure your web browser is up to date and your camera is properly configured. Some browsers also require you to access Snap! through HTTPS to use the camera. Plase replace the \"http://\" part of the address in your browser by \"https://\" and try again.": "",
+    "Please make sure your web browser is up to date and your camera is properly configured. Some browsers also require you to access Snap! through HTTPS to use the camera. PPleaselase replace the \"http://\" part of the address in your browser by \"https://\" and try again.": "",
```

Comment:
this seems to be another mis-spelling :). Don't worry, though, I'll take care of it. Tnx